api-js/src/organization/objects/organization-summary.js

```diff
@@ -23,12 +22,12 @@ export const organizationSummaryType = new GraphQLObjectType({
 
         const categories = [
           {
-            name: i18n._(t`pass`),
+            name: i18n._('pass'),
```

Comment:
After removing the `t` macro, then running `npm run lin-clean` the translations are then removed from `.po` files.

![Screenshot from 2021-01-21 07-09-14](https://user-images.githubusercontent.com/7891920/105343179-acbad400-5bb7-11eb-92fa-667773170b6c.png)

Reply:
Good catch! I've updated it to use `defineMessage` to define a translatable string that lingui recognizes and extracts. Take another look and tell me what you think.